app/models/audio_version_template.rb

```diff
@@ -87,7 +87,8 @@ def audio_file_content_type_errors(audio_file)
     if content_type.present?
       tpl_type = content_type.split('/').first
       file_type = audio_file.content_type.try(:split, '/').try(:first)
-      if content_type == 'audio/mpeg' && audio_file.content_type != 'audio/mpeg'
+      if content_type == AudioFile::MP3_CONTENT_TYPE && \
```

Comment:
I don't think you need that backslash, the `&&` is enough to indicate to the interpreter that the expression continues to the next line.